readme.md

```diff
@@ -284,7 +284,11 @@
 - [Open Source Society University](https://github.com/open-source-society/computer-science)
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming)
 - [Static Analysis & Code Quality](https://github.com/mre/awesome-static-analysis)
+
+
+## Information Science
```

Comment:
Creating a category should be a separate PR after this one. You also need to add it to the `Contents` section.